src/Nanoid/Nanoid.cs

```diff
@@ -1,107 +1,278 @@
+                public const string Symbols = "_-";
+            }
+
+            /// <summary>
+            /// All digits [0, 9].
+            /// </summary>
+            public const string Digits = "0123456789";
+
+            /// <summary>
+            /// English hexadecimal with lowercase characters.
+            /// </summary>
+            public const string HexadecimalLowercase = Digits + "abcdef";
+
+            /// <summary>
+            /// English hexadecimal with uppercase characters.
+            /// </summary>
+            public const string HexadecimalUppercase = Digits + "ABCDEF";
+
+            /// <summary>
+            /// Lowercase English alphabet letters.
+            /// </summary>
+            public const string LowercaseLetters = "abcdefghijklmnopqrstuvwxyz";
+
+            /// <summary>
+            /// Uppercase English alphabet letters.
+            /// </summary>
+            public const string UppercaseLetters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
+
+            /// <summary>
+            /// Lowercase and uppercase English alphabet letters.
+            /// </summary>
+            public const string Letters = LowercaseLetters + UppercaseLetters;
+
+            /// <summary>
+            /// Lowercase English alphabet letters and digits.
+            /// </summary>
+            public const string LowercaseLettersAndDigits = Digits + LowercaseLetters;
+
+            /// <summary>
+            /// Uppercase English alphabet letters and digits.
+            /// </summary>
+            public const string UppercaseLettersAndDigits = Digits + UppercaseLetters;
+
+            /// <summary>
+            /// English alphabet letters and digits.
+            /// </summary>
+            public const string LettersAndDigits = Digits + LowercaseLetters + UppercaseLetters;
+
+            /// <summary>
+            /// English alphabet letters and digits without lookalikes: 1, l, I, 0, O, o, u, v, 5, S, s, 2, Z.
+            /// </summary>
+            public const string NoLookAlikes = SubAlphabets.NoLookAlikeDigits + SubAlphabets.NoLookAlikeLetters;
+
+            /// <summary>
+            /// English alphabet letters and digits without lookalikes (1, l, I, 0, O, o, u, v, 5, S, s, 2, Z)
+            /// and with removed vowels and the following letters: 3, 4, x, X, V.
+            /// This list should protect you from accidentally getting obscene words in generated strings.
+            /// </summary>
+            public const string NoLookAlikesSafe = SubAlphabets.NoLookAlikeSafeDigits + SubAlphabets.NoLookAlikeSafeLetters;
+
+            /// <summary>
+            /// The default alphabet used by Nanoid. Includes ascii digits, letters and the symbols '_' and '-'.
+            /// </summary>
+            public const string Default = SubAlphabets.Symbols + LettersAndDigits;
+        }
+
+        /// <summary>
+        /// The default size of a Nanoid.
+        /// </summary>
+        public const int DefaultIdSize = 21;

```

Comment:
`DefaultIdSize` is now public for ease of reference. For example I needed this constant when using the `MaxLength` attribute on a nanoid.